amp/amp-theme/index.html.twig

```diff
@@ -35,7 +35,7 @@ Available variables:
       {% if media.renditions is iterable %}
           {% for rendition in media.renditions %}
               {% if rendition.name == '770x515' %}
```

Comment:
it does not matter bcs we have all renditions anyway. I would say it could be smaller. Leave it for now.